package/autoyast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Jun  7 13:06:43 UTC 2021 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Abort the installation importing the users sections from
+  the profile fails (related to jsc#PM-2620).
```

Comment:
As discussed _offline_, maybe we should have another reference for this and other changes. Anyway, since seems that we have been using it lately, let's go ahead if nobody is against.